mlrun/config.py

```diff
@@ -120,6 +120,14 @@
         "projects": {
             "summaries": {
                 "cache_interval": "30",
+                "feature_gates": {
```

Comment:
Since you put feature gates on all types of counters, please also add one for `"functions"` (which is handled in `_calculate_functions_counters()`.

Reply:
I don't see anywhere that uses that method, but sure